.pylintrc

```diff
@@ -6,13 +6,14 @@
 # C1001: Old-style class defined.
 # D100:  Missing docstring in public module.
 # E1101: %s %r has no %r member.
+# MC0001: mccabe plugin
```

Comment:
@deshraj Yeah, now we don't need this. I just tried to disable a pylint plugin `mccabe`. I have done that using the `.landscape.yml` file.